.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,13 @@
+<!-- PR title format: <type>(<issue-number>): <description>
+     Example: chore(272): add plan documentation system
+     Types: feat, fix, chore, docs, refactor, test, perf -->
+
+## Summary
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)